google/auth/external_account_authorized_user.py

```diff
@@ -290,23 +291,38 @@ def refresh(self, request):
     def _make_sts_request(self, request):
         return self._sts_client.refresh_token(request, self._refresh_token)
 
+    @_helpers.copy_docstring(credentials.Credentials)
+    def get_cred_info(self):
+        if self._cred_file_path:
+            return {
+                "credential_source": self._cred_file_path,
+                "credential_type": "external account authorized user credentials",
```

Comment:
is there no way to get a principal for external user?

Reply:
It requires a network call to introspect endpoint, this feature is only available in gcloud. In gcloud when we do auth login, it calls the introspect endpoint to get the account then saves the account to core/account property, I feel this is sort of a requirement for gcloud to know the account for each cred, but not auth lib. If BYOID team adds this support in auth lib some day, we can include it here.